### DISCUSSION TITLE
Pre-filled fields which are disabled and the submit event

### DISCUSSION THREAD

**Discussion by Nickvda** [react-hook-form/react-hook-form#2751] [Q&A] [answered]
Greetings!

In my current case I have a number of fields which are disabled on a specific condition. All fields are
pre-filled and all need to come through the submittal. Is there a way to access the disabled-field pre-values?

Or am I going to have to be crafty and use hidden fields for this?

Thanks in advance and for reading 👍

**Comment by bluebill1049** [comment, marked as answer]
from the sound of it, I think you will need hidden input or custom register.

**Comment by Nickvda** [comment]
Thanks for the fast response!

Does that mean that I will have to back-up a disabled field with a hidden field at all times?
And if I bind these 2 fields to the same name (1 disabled, 1 hidden) it won't conflict?

**Reply by bluebill1049** [reply]
why not use read-only?

**Reply by Nickvda** [reply]
Haha, because I had honestly not thought about that! And that works like a complete charm.. Thanks! 👍
